zilliqa/src/api/types/zil.rs

```diff
@@ -291,7 +289,7 @@ impl GetTxResponse {
                     })
                     .collect(),
                 success: receipt.success,
-                accepted: receipt.accepted,
+                accepted: receipt.accepted.unwrap_or(false),
```

Comment:
This suggests that ZQ1 returns `accepted: false` for:
* Eth transactions
* Scilla contract creations

Have we double-checked that is definitely the case?

Reply:
No, my assumption was that a missing accepted field was an error state.
We could do `accepted: receipt.accepted.unwrap_or(receipt.success)` instead?
The reason for doing this is was that we're always returning an accepted value to match ZQ1, so we could also just not.

Reply:
I guess my question is what does ZQ1 return as `accepted` for transactions which don't have a natural value? (for context - `accepted` in Scilla-land means "the Scilla contract you sent ZIL to accepted the funds")

Reply:
#1666 Suggests that ZQ1 returns false where the previous ZQ2 implementation would have returned nothing since receipt.accepted == none. But I don't necessarily know the details of transaction 4964bf50c8f59... so I might be misunderstanding. Does that match what you thing that transaction does, @chetan-zilliqa